src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/PartyFlagsMappingTest.java

```diff
@@ -355,32 +350,36 @@ void urgentCase(String urgentCase) {
     }
 
 
-    @DisplayName("adjournCaseInterpreterLanguage Parameterised Tests")
-    @ParameterizedTest
-    @ValueSource(strings = {"spanish", "french"})
-    void adjournCaseInterpreterLanguage(String interpreterLanguage) {
+    @DisplayName("adjournCaseInterpreterLanguage should return party flag mapping if interpreter required")
+    @Test
+    void adjournCaseInterpreterLanguage() {
         SscsCaseData caseData = SscsCaseData.builder()
```

Comment:
Minor - double check indents from here down. I prefer single indent across the board. I'm sure it's personal preference but to me, uniformity is easier to read than trying to group a set of indented extensions directly under their parent.

Reply:
updated